turbinia/e2e/googleclouddisk.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# This Turbinia e2e test will test the googleclouddisk functionality.
+# It will try to analyse the disk 'test-disk2' in zone 'us-central1-a'.
+
+
+TURBINIA_CLI="turbiniactl"
+MAIN_LOG="main.log"
+STATUS_LOG="status.log"
+STATS_LOG="stats.log"
+DETAIL_LOG="jobdetails.log"
+OUT_TGZ="jobresults.tgz"
+DISK="test-disk2"
+ZONE="us-central1-a"
+
+echo "Creating GCE test disk to use in e2e test"
+gcloud compute disks create $DISK --image=$DISK --zone=$ZONE
+
+echo "Executing googlecloudisk e2e test....this takes ~60 minutes!"
+$TURBINIA_CLI -L $MAIN_LOG -a -w googleclouddisk -d $DISK -z $ZONE &
+
+# Sleep here to make sure Turbinia can get started and we have a main.log file to tail.
+sleep 60
+
+# Parse turbiniactl output and wait for job ID.
+JOB_ID_LINE=`grep -m 1 "Creating request" <(tail -f $MAIN_LOG)`
+JOB_ID=`echo "$JOB_ID_LINE" | grep -oP "request \K\w+(?= with)"`
```

Comment:
Per above, assuming this would be `REQUEST_ID` instead of `JOB_ID`

Reply:
ack